testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1718,6 +1718,41 @@ public void testAddColumn()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testAddColumnWithComment()
+    {
+        if (!hasBehavior(SUPPORTS_ADD_COLUMN)) {
+            assertQueryFails("ALTER TABLE nation ADD COLUMN test_add_column_with_comment bigint", "This connector does not support adding columns");
```

Comment:
The `nation` table seems to be the table most used in the `BaseConnectorTest` for such negative tests. Creating a temp table would need extra checks to see whether a table can be created and this would only reduce the readability of the tests. I'd rather stick with the current form of the negative test.

Reply:
see https://github.com/trinodb/trino/pull/11493#discussion_r828184757. the current impl seems fine because of default value of new TestingConnectorBehaviour.